packages/schematics/angular/web-worker/files/worker-tsconfig/tsconfig.worker.json.template

```diff
@@ -1,4 +1,5 @@
-/* To learn more about this file see: https://angular.io/config/tsconfig. */
+/* To learn more about Typescript configuration file: https://www.typescriptlang.org/docs/handbook/tsconfig-json.html. */
+/* To learn more about Angular compiler options: https://angular.dev/reference/configs/angular-compiler-options. */
```

Comment:
Workers are not part of the Angular compilation. 

```suggestion
```

Reply:
Nice catch, done in c1ec6c1aae0ec56d9d9a250f624f66ac8c0bbf2c